docs/schemas/profile/rules.json

```diff
@@ -0,0 +1,209 @@
+                },
+                "all": {
+                    "$ref": "#/definitions/conditionArray"
+                }
+            }
+        },
+        "name": {
+            "$id": "#/properties/name",
+            "default": {},
+            "description": "A way of naming your rules, allowing them to be easily identifiable in Rule Results. This is usually of type String, but could also be Object, Array, or Number. Note that the name need not be unique, and that it has no impact on execution of the rule.",
+            "examples": [
+                "My Rule Name"
+            ],
+            "title": "The Name Schema",
+            "type": "string"
+        }
+    },
+    "definitions": {
+        "conditionArray": {
+            "type": "array",
+            "title": "Condition Array",
+            "description": "An array of conditions with a possible recursive inclusion of another condition array.",
+            "default": [],
+            "items": {
+                "anyOf": [
+                    {
+                        "$ref": "#/definitions/conditions"
+                    },
+                    {
+                        "$ref": "#/definitions/condition"
+                    }
+                ]
+            }
+        },
+        "condition": {
+            "type": "object",
+            "title": "Condition",
+            "description": "Rule conditions are a combination of facts, operators, and values that determine whether the rule is a success or a failure. The simplest form of a condition consists of a fact, an operator, and a value. When the engine runs, the operator is used to compare the fact against the value. Sometimes facts require additional input to perform calculations. For this, the params property is passed as an argument to the fact handler. params essentially functions as fact arguments, enabling fact handlers to be more generic and reusable.",
+            "default": {
+                "fact": "my-fact",
+                "operator": "lessThanInclusive",
+                "value": 1
+            },
+            "examples": [
+                {
+                    "fact": "gameDuration",
+                    "operator": "equal",
+                    "value": 40.0
+                },
+                {
+                    "value": 5.0,
+                    "fact": "personalFoulCount",
+                    "operator": "greaterThanInclusive"
+                },
+                {
+                    "fact": "product-price",
+                    "operator": "greaterThan",
+                    "path": "$.price",
+                    "value": 100.0,
+                    "params": {
+                        "productId": "widget"
+                    }
+                }
+            ],
+            "required": [
+                "path",
+                "operator",
+                "value"
+            ],
+            "properties": {
+                "operator": {
+                    "type": "string",
```

Comment:
I think the value of operator are not good : https://github.com/santalvarez/python-rule-engine/blob/main/docs/operators.md

we must use `not_equal` instead of `notEqual` for example

Reply:
Good catch, thxs!